pom.xml

```diff
@@ -521,7 +501,7 @@
                 </property>
             </activation>
             <properties>
-                <docker.skip>true</docker.skip>
+                <!-- <docker.skip>true</docker.skip> -->
```

Comment:
I don't even know when this is used. If you don't use it regularly, I suggest removing the whole profile.